app/views/admin/user.scala.html

```diff
@@ -266,6 +269,45 @@ <h1>Feedback</h1>
                 AdminUser('@user.username');
                 $('#label-table').dataTable();
             });
+
+            // Displays checkbox for user volunteer status
+            var serviceHoursUser = @UserRoleTable.getCommunityService(UUID.fromString(user.userId));
```

Comment:
This JavaScript code should all live in `Admin.User.js` instead of in the HTML file!